.github/workflows/ci.yml

```diff
@@ -78,9 +91,11 @@ jobs:
 
     - name: Test
       id: test
-      env:
-        COVERAGE_FILE: ".coverage.${{ matrix.platform }}.${{ matrix.python-version }}"
-      run: tox -e py --installpkg dist/briefcase-*.whl
+      run: |
+        RUNNER_OS=$(cut -d- -f1 <<< ${{ matrix.platform }})
+        RUNNER_VERSION=$(cut -d- -f2 <<< ${{ matrix.platform }})
+        COVERAGE_FILE=".coverage.${RUNNER_OS}.${RUNNER_VERSION}.${{ matrix.python-version }}" \
+          tox -e py --installpkg dist/briefcase-*.whl
```

Comment:
This nonsense is necessary because `coverage` doesn't just do a substring match on the coverage file names; it adds a period to the end of the substring match...so, the files literally need to start with `.coverage.ubuntu.`, for instance.